.pre-commit-config.yaml

```diff
@@ -24,3 +24,14 @@ repos:
     rev: 'v0.0.269'
     hooks:
       - id: ruff
+  # Hook to enforce a consistent markdown linewrap policy
```

Comment:
I actually dislike line wrapped markdown. The preference comes from our Manubot project for writing collaborative manuscripts with markdown, where we [recommend](https://github.com/manubot/rootstock/blob/main/USAGE.md#manubot-markdown) one sentence per line. I would be okay enforcing that with a hook.